stix_shifter_modules/elastic_ecs/stix_translation/json/to_stix_map.json

```diff
@@ -232,6 +468,816 @@
         "object": "proc",
         "references": "user"
       }
+    ],
+    "email": [
+      {
+        "key": "email-addr.value",
+        "object": "email"
+      },
+      {
+        "key": "email-addr.belongs_to_ref",
+        "object": "email",
+        "references": "user"
+      }
     ]
+  },
+  "agent": {
+    "name": {
+      "key": "software.name"
+    },
+    "type": {
+      "key": "software.vendor"
+    },
+    "version": {
+      "key": "software.version"
+    }
+  },
+  "cloud": {
+    "account": {
+      "id": {
+        "key": "x_ecs_cloud.account_id",
+        "cybox": false
+      }
+    },
+    "availability_zone": {
+      "key": "x_ecs_cloud.availability_zone",
+      "cybox": false
+    },
+    "instance": {
+      "id": {
+        "key": "x_ecs_cloud.instance_id",
+        "cybox": false
+      },
+      "name": {
+        "key": "x_ecs_cloud.instance_name",
+        "cybox": false
+      }
+    },
+    "machine": {
+      "type": {
+        "key": "x_ecs_cloud.machine_type",
+        "cybox": false
+      }
+    },
+    "provider": {
+      "key": "x_ecs_cloud.provider",
+      "cybox": false
+    },
+    "region": {
+      "key": "x_ecs_cloud.region",
+      "cybox": false
+    }
+  },
+  "container": {
+    "id": {
+      "key": "x_ecs_container.id",
+      "cybox": false
+    },
+    "image": {
+      "name": {
+        "key": "x_ecs_container.image_name",
+        "cybox": false
+      },
+      "tag": {
+        "key": "x_ecs_container.image_tag",
+        "cybox": false
+      }
+    },
+    "labels": {
+      "key": "x_ecs_container.labels",
+      "cybox": false
+    },
+    "name": {
+      "key": "x_ecs_container.name",
+      "cybox": false
+    },
+    "runtime": {
+      "key": "x_ecs_container.runtime",
+      "cybox": false
+    }
+  },
+  "dll":{
```

Comment:
According to the docs, dll should have nested code_signature, hash, and pe objects.